src/pages/ReimbursementAccount/BankAccountPlaidStep.js

```diff
@@ -1,156 +0,0 @@
-import {useIsFocused} from '@react-navigation/native';
```

Comment:
Was this file deleted? Seems like Github is showing that the contents got deleted but not the file itself